content/en/infrastructure/cloud_cost_management.md

```diff
@@ -16,6 +16,16 @@ To use Cloud Cost Management, you must have an AWS account with access to Cost U
  
 **Note:** Data can take up to 48 to 72 hours after setup to stabilize in Datadog.
  
+### Configuration
+
+To configure a data source, access the Cloud Costs product by clicking **Infrastructure > Cloud Costs** using the main navigation.
+
+If there are no configured data sources, you are redirected to a setup page with instructions for generating and enabling access to your AWS account's [Cost and Usage Report (CUR)][5].
+
+{{< img src="infrastructure/cloudcost/cloud_cost_data_source_config.png" alt="Configure the AWS data source for Cloud Costs"  >}}
+
+Once configured, data sources may be viewed and modified by accessing the **Settings tab** at the top of the page.
+
```

Comment:
Might be useful here to mention that we currently support only a single data source, i.e. a single account at a time (and maybe also remind the reader that for now its only AWS).